### ISSUE TITLE
ensure abstract in NAME section, fix generated manpage whatis entries

### ISSUE THREAD

**Issue by fschlich** [rustyconover/net-amazon-s3#108]
Debian generates manpages from POD. A manual page should start with a NAME section, which lists the program name and a brief description. The NAME section is used to generate a database that can be queried by commands like apropos and whatis. For net-amazon-s3, this fails in a few cases because the NAME section only contains the name section and no abstract.

In Debian, we're applying the following patch:

```
--- a/lib/Net/Amazon/S3/ACL/Set.pm
+++ b/lib/Net/Amazon/S3/ACL/Set.pm
@@ -124,7 +124,7 @@ __END__
 
 =head1 NAME
 
-Net::Amazon::S3::ACL::Set
+Net::Amazon::S3::ACL::Set - Representation of explicit ACL
 
 =head1 VERSION
 
diff --git a/lib/Net/Amazon/S3/Response.pm b/lib/Net/Amazon/S3/Response.pm
index acbe699a..4d0d3a06 100644
--- a/lib/Net/Amazon/S3/Response.pm
+++ b/lib/Net/Amazon/S3/Response.pm
@@ -251,7 +251,7 @@ __END__
 
 =head1 NAME
 
-Net::Amazon::S3::Response
+Net::Amazon::S3::Response - Behaviour common to most S3 responses
 
 =head1 VERSION
```

I'd send a pull request, but these sections seem to be auto-generated and it's not obvious to me why the ABSTRACT already present in the file isn't used. This affects a few more than just these two modules...

**Comment by happy-barney** [comment]
Thanks for report, looks like few files (including mentioned) has `Abstract` instead of `ABSTRACT`

**Comment by happy-barney** [comment]
Solved in #116 / release v0.99
